extension/controller/backgroundScript.js

```diff
@@ -33,6 +33,20 @@ let translationRequestsByTab = new Map();
 let outboundRequestsByTab = new Map();
 const translateAsBrowseMap = new Map();
 
+// set data collection accordingly with what the user set
+browser.storage.local.get("telemetryCollectionConsent").then(item => {
+  if (Object.keys(item).length !== 0) {
+    settings.uploadTelemetry = item.telemetryCollectionConsent;
+  }
+});
+
+browser.storage.local.get("errorCollectionConsent").then(item => {
+  if (Object.keys(item).length !== 0) {
+    Sentry.getCurrentHub().getClient()
+      .getOptions().enabled = item.errorCollectionConsent;
```

Comment:
The Telemetry disable above works, but not this (found this command in a couple of references). `Sentry.init` is called in three classes, so will we need to send three messages?  Maybe there's another way to disable it? @eu9ene

Reply:
Sentry is initialized separately in bgScript, contenScript(mediator) and web worker. Since race conditions are possible, we should add message flow to notify mediator and web worker to disable sentry there in the same way.